### ISSUE TITLE
Update to use MP Health 3.0

### ISSUE THREAD

**Issue by Emily-Jiang** [OpenLiberty/guide-kubernetes-microprofile-health#98]

**Comment by pgunapal** [comment]
NEW changes in mpHealth-3.0:

- The `HealthCheckResponseBuilder.state(Boolean UP)` method was also renamed to `HealthCheckResponseBuilder.status(Boolean UP)` for HealthCheckResponse deserialization compatibility, where the JSON health check response string can now be deserialized into an HealthCheckResponse object.
- The deprecated `@Health` qualifier was removed, and users should use the `@Liveness` or `@Readiness` qualifiers in their HealthCheck implementations, as appropriate.

The following guides need to be updated to be compatible with mpHealth-3.0:
- https://openliberty.io/guides/getting-started.html#updating-the-source-code-without-restarting-the-server
- https://openliberty.io/guides/kubernetes-microprofile-health.html#deploying-the-microservices
- https://openliberty.io/guides/microprofile-health.html#running-the-tests

Basically, in the above guides, the following files use the `state()` method, and should be updated to use the new `status()` method.
 - SystemLivenessCheck.java
 - InventoryLivenessCheck.java

Also, the deprecated `@Health` annotation is removed, so please remove any occurrences of it in any of the guides.

**Comment by gkwan-ibm** [comment]
close
- the [guide](https://openliberty.io/guides/kubernetes-microprofile-health.html) was updated
